template/templates/README.md.tpl

```diff
@@ -0,0 +1 @@
+This is README.md for templates directory
```

Comment:
Current information is not sufficient, but i believe we should keep README.md files to provide developer(without much ACK knowledge) some context about these directories.

Reply:
That's fine, but these README's aren't doing that. Plus this is another thing we now need to keep consistent across 50+ repositories. I'd rather we have contributor documentation on the website, where we can manage it all from one place, rather than replicating it everywhere